bin/helpers/timeComponents.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
+const { isUndefined } = require('./utils');
+
+let sessionTimes = {
+  referenceTimes: {
+    absoluteStartTime: Date.now()
+  }, // Absolute times which needs to be used later to calculate logTimes
+  logTimes: {}, // Time Difference in ms which we need to push to EDS
+};
+
+const initTimeComponents = () => {
+  sessionTimes = {referenceTimes: {absoluteStartTime: Date.now()}, logTimes: {}};
```

Comment:
Why are we duplicating it? It is already initialized on line number 5?

Reply:
Which is fine for current case. But going forward we may need explicit initialization hence keeping this.

Reply:
Can we please do it now?

Reply:
explicitly initializing in like 18 run.js.